docs/ref/forms/fields.txt

```diff
@@ -562,7 +562,9 @@ For each field, we describe the default widget used if you don't specify
 
     .. attribute:: step_size
 
-        Limit valid inputs to an integral multiple of ``step_size``.
+        Limit valid inputs to an integral multiple of ``step_size``. If
+        ``min_value`` is provided as well, then this value is added as offset
```

Comment:
```suggestion
        ``min_value`` is also provided it's added as an offset
```

Reply:
If this change is acceptable (or whatever is easiest to read) then the same would need to be applied below.